src/better_lane_follow/src/LineDetect.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Created by: Raad Khan
+ * Created On: April 23, 2017
+ * Description: Takes in an image feed and generates lane lines.
+ * Usage:
+ * Subscribes to:
+ * Publishes to:
+ */
+
+#include <LineDetect.h>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/objdetect/objdetect.hpp>
+
+LineDetect::LineDetect() : initalLineDetectThreshold(200),
```

Comment:
These are simple integers, so I don't see a need for an initialisation list - these are generally used if you're constructing more complicated members of a class. Also, `initalLineDetectThreshold(200)` is misspelt.